### ISSUE TITLE
Excessive Jittering

### ISSUE THREAD

**Issue by digitallemming** [LukeRoss00/gta5-real-mod#33]
I'm not entirely sure what's wrong but it seems much more shaky then I've scene in other videos. I've tried both the Low and High end specs. Here is a short clip you can see it is shaking even while just looking straight or walking. https://www.youtube.com/watch?v=8tG9R5B3niE&feature=youtu.be

Specs are
Titan XP Limited Edition
Ryzen 2700x
32gb DDR4
Asrock Taichi 470x

Playing on Oculus Rift, I have a Rift S but have not tried it yet.

My monitor is 3440x1440 but I'm not sure that would effect anything. The GTA window appears to be opening at the proper 1080x1080 square window but the in game settings while playing show a different resolution. My GPU is entered at the bottom of the settings folder and it is set to "Read Only"

I'm not sure if it's normal but the window showing the game on the PC is alternating left eye right eye as well. Also if there is a better way to record video I would like too know. The recording is the oculus mirror and is shaky and the game window is alternating frames.

**Comment by LukeRoss00** [comment]
Your specs should obviously be more than enough to run the mod smoothly. The monitor is fine too, the only important thing there is that your desktop resolution is high enough to contain the game window. BTW, it's normal that the game window on screen should show the eyes alternating; what's abnormal is the jerkiness in the headset. Neither the headset nor the Oculus mirror should be jittery.

A small number of other users seem to have the same problem as you, but for now I can't see any common pattern. What I can tell you though is that on my own PC similar symptoms appeared randomly one day after a reboot, and to make them go away I had to uninstall a lot of Asus crapware that my motherboard had [sneaked in](https://www.techpowerup.com/248827/asus-z390-motherboards-automatically-push-software-into-your-windows-installation). The main culprit in my case was some unwanted RGB lighting "utility" which had suddenly started to malfunction, but of course that's only anecdotal evidence.

However, I would start by looking at the usual suspects, like the various pieces of overlay software that might be interfering with frame timing/presentation.

If you cannot get the mod to run smoothly after terminating/disabling overlays and bloatware, please write again and we'll keep investigating!

**Comment by digitallemming** [comment]
Thanks for getting back to me, I'll experiment with that for the time being and see if I can get things going.

**Comment by cercata** [comment]
I had some Jittering the day RDR2 launched, I guess the launcher is quite intrusive.

Also some days I have crashes, and I have been seeing in forums and reddit that it happens in the vanilla version since they launched the new launcher 1-2 months ago.

**Comment by satyaloka93** [comment]
More shaky here also, with more pixelation. I just opened an issue, maybe something with AMD CPUs/chipsets? Ryzen 5 3600, B450 motherboard here.

**Comment by digitallemming** [comment]
Hey LukeRoss if you right click GTAV in steam go to "properties" and "set launch options" what is the display size you have to set launch there? My game was opening up in a strange window size regardless of what the settings file was saying https://imgur.com/AChF0o0 see how the edges are cut. This was me attempting to play normally without the mod. I'm wondering it this had something to do with it.

**Comment by LukeRoss00** [comment]
The window looks right, although there's only so much that one can tell from a screenshot. It seems strange however that the card name should contain "COLLECTORS EDITION"...

Perhaps try this: delete the `Documents\Rockstar Games\GTA V\settings.xml` file completely, and also delete the binary file `Documents\Rockstar Games\GTA V\Profiles\<8 hex digits>\pc_settings.bin`. Uninstall the mod by deleting the files it created when you extracted the .rar file. Run GTA V once, and it will recreate the settings files, then quit after making the menu changes described in the README (Targeting Mode, etc.).

At this point you'll have the correct card name that GTA needs in the newly created `settings.xml` file. Reinstall the mod, and start from the low settings file, taking care to put in your correct card name exactly.

If the shakiness is gone from the headset and the Oculus mirror, you can try to move toward the high settings.

**Comment by digitallemming** [comment]
Thanks, yeah that's how I did it. I started with a fresh install, ran the game to change the suggested settings, backed up the settings.xml, then added your settings .xml and then copy the GPU from the  original settings to your settings. The card is the Collectors Edition Stars wars one so that must be why it pops up. I'm going to try a 1080ti and see if that helps. If not it's all good, I appreciate the help but don't want to bother you too much for something you did for free.
